### PR TITLE
Bookstore: Adding styling to the project

### PR DESCRIPTION
**Pull request by Ellyboi**
**Below requirement implemented:**

- Add missing JSX elements to the project
- The final design almost match Zeplin's design but can not be 100%
- I used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Navigation from './components/Navigation';
 function App() {
   return (
     <>
-      <div className="booknav-container">
+      <div className="booknav-container flex-con flex-center">
         <Navigation />
         <Routes>
```
Comment (TadesseDev) [inline src/App.js:12]: - [ ] Impressive job already @Ellyboi, But I see there is a minor miss understanding of the design. Please notice he pard that is shown behind the design is not part of the webpage. We need to remove that part and have the full page without the background of the template. Please update your design accordingly. 
- [ ] also, Your circles seem to be a bit thick compared with the given design, Please consider updating more thickness to it. This is just to make sure we have an exact copy of the given design.

### Your design 
image
### Original/Expected design
image

File: src/components/AddBook.jsx [hunk 2]
```diff
@@ -28,8 +29,8 @@ const AddBook = () => {
 
   return (
     <>
-      <div className="form-container">
-        <h2>Add a new Book</h2>
+      <div className={styles.formContainer}>
+        <h2 className={styles.formTitle}>Add a new Book</h2>
         <form onSubmit={handleSubmit}>
           <input
```
Comment (TadesseDev) [inline src/components/AddBook.jsx:35]: - [ ] Dear @Ellyboi, for the add book section, Your design seems to not follow the structure of the rest of the page. In the original design, this section follows the left and right margins of the list of books. Please take a look at the difference and make appropriate changes. Remember as a frontend engineer we are expected to deliver the exact copy of the original design. 

### Your design 
image
### Original/expected design 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Ellyboi 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitFlow  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Please try to separate your code into different levels of commits. Remember your commits are going to help you look back and debug your code. It's a good idea to commit after every visible change.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello @Ellyboi  😃 

You're doing a great job on your project. We're proud of you. Good job so far! 

It's OK to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of GitHub Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] As requested by the previous Code Reviewer, I see there is a minor miss understanding of the design. Please notice the part that is shown behind the design is not part of the webpage. We need to remove that part and have the full page without the background of the template. Please update your design accordingly. 

### As you can see in the recording:
https://github.com/Ellyboi/bookstore/assets/83048208/6e40a836-3091-4530-aced-3ea1299a2e8d



_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
Hi @Ellyboi, 🤩🔥 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Great job on the functionaliy for design 💯
- [ ] The styles match the wireframe ✨✨
- [ ] Professional README 🔥
- [ ] Followed Gitflow 🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**